### DISCUSSION TITLE
CC selector with encoder?

### DISCUSSION THREAD

**Discussion by eurorackworld** [shanteacontrols/OpenDeck#218] [General]
Hi,

Would it be possible to do something like this?
<img width="204" alt="image" src="https://github.com/shanteacontrols/OpenDeck/assets/29734704/661cba1d-8c6a-439c-ad28-ce53d4aed53d">

Which control would make it possible to have a CC selected? Limit an analog control via the upper limit to 11 works, but as analog control doesn't have 'clicks' its hard to note when the value changed. Using a encoder sounds more logical, but it is not possible to set those same limits right? As the encoder now goes from 0-127 al the time, although the selecting and 'click' works fine (each 'click' increases or decreases the value by one), so an encoder is not possible to use?

It should then be a rotary switch (with 12 'buttons')? Although it feels weird as this requires quite some config and work while an encoder with limited values would sound more logical.

Do I miss something here?

**Comment by paradajz** [comment]
> Using a encoder sounds more logical, but it is not possible to set those same limits right? 

You're right, the feature is currently not implemented.

**Comment by paradajz** [comment]
Now implemented on master branch.
